src/index.html

```diff
@@ -11,7 +11,190 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
```

Comment:
better multi-line for readability
even-though it's only two attributes, but with one long it's not that readable
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="post"
    >
```